src/fontstash.h

```diff
@@ -1007,20 +1079,34 @@ static void fons__blur(FONScontext* stash, unsigned char* dst, int w, int h, int
 }
 
 static FONSglyph* fons__getGlyph(FONScontext* stash, FONSfont* font, unsigned int codepoint,
-								 short isize, short iblur)
+								 short isize, short iblur, int oversampleH, int oversampleV)
 {
 	int i, g, advance, lsb, x0, y0, x1, y1, gw, gh, gx, gy, x, y;
 	float scale;
 	FONSglyph* glyph = NULL;
 	unsigned int h;
 	float size = isize/10.0f;
+	float subX, subY;
+	float recipH, recipV;
 	int pad, added;
 	unsigned char* bdst;
 	unsigned char* dst;
 
 	if (isize < 2) return NULL;
 	if (iblur > 20) iblur = 20;
-	pad = iblur+2;
+	if (oversampleH < 1) oversampleH = 1;
+	if (oversampleV < 1) oversampleV = 1;
+
+	// Only use extra 1px "leak" padding if blurring (otherwise it breaks oversampling)
```

Comment:
We always need 2px around the actual pixel data to render the glyphs correctly. One pixel inside the rendered quad and one pixel outside. The reason is that otherwise rotated glyphs will pick up neighbour data.